### PR TITLE
Updated KR translations

### PR DESCRIPTION
**Pull request by psp0**
😵‍💫(I don't know how to deal with white-space. but If you only allow changes, tell me. Then I'll try to fix it as much I can.)

### CODE REVIEW

**Comment by RaenonX** [comment]
Rebase all then it should be good

**Comment by psp0** [comment]
I just compared it after overwriting.

File: messages/ui-kr.json [hunk 1]
```diff
@@ -450,7 +450,7 @@
         "Skill": "메인 스킬"
       }
     },
-    "DrowsyPower": "에너지 x 스코어",
+    "DrowsyPower": "에너지 x 스코어 x **추가 졸음 파워 보너스(%)**",
```
Comment (RaenonX) [inline messages/ui-kr.json:453]: Remove the `**` as the text won't be formatted

**Review by RaenonX — CHANGES_REQUESTED** [review]

File: messages/ui-kr.json (hunk 1 above)
Comment (psp0) [reply messages/ui-kr.json]: ah I thought it'll be able to highlight.😅

**Review by RaenonX — APPROVED** [review]

**Pull request merged**